react/react1/week1/to-do/src/Components/ToDoList.jsx

```diff
@@ -0,0 +1,35 @@
+
+import ToDoItem from './ToDoItem'
+import ToDoList from './ToDoHeader'
+const listArray = [
+  {  task: "Get out of bed", deadLine: "Wed Sep 13 2017" },
+  { task: "Brush teeth", deadLine: "Thu Sep 14 2017" },
+  { task: "Eat breakfast", deadLine: "Fri Sep 15 2017" }
+];
+
+function ToDo() {
+  return (
+    <div className="header">
+      <ToDoList/>
+      <ItemKey/>
+      </div>  
+  );
+}
+function ItemKey(){
+return (
+    <ul className="list">
+{listArray.map((eachList,i)=> {
+ return <TaskItem taskItem={eachList} key={i}/>
+})}
+    </ul>
+)
+}
+function TaskItem({taskItem}){
+return (
+  <div className="eachitem">
+<ToDoItem task={taskItem.task}
```

Comment:
You can pass `tastItem={taskItem}` and access its props inside `ToDoItem.jsx` component. :)

Reply:
Yes,in class we had practice exactly this way so just made it here but your suggestion keeps more sense in this logic,I will change as per your advice,thank you.